### PR TITLE
New module l10n_br_hr

### PR DESCRIPTION
**Pull request by sadamo**

### CODE REVIEW

**Comment by renatonlima** [comment]
Hello @sadamo 

Please check travis pylint. Thank you for your contribute!

**Comment by renatonlima** [comment]
It would be interesting add some tests in this module.

File: l10n_br_hr/model/hr_employee.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+from datetime import datetime
+from openerp.tools import DEFAULT_SERVER_DATE_FORMAT
+from openerp.exceptions import Warning as UserError
+
+
+class HrEmployee(models.Model):
+    _inherit = 'hr.employee'
+
+    @api.constrains('pis_pasep')
+    def _validate_pis_pasep(self):
```
Comment (renatonlima) [inline l10n_br_hr/model/hr_employee.py:16]: Maybe this function should be in fiscal.py in l10n_br_base/tools/ to be reusable.

**Comment by rvalyi** [comment]
@sadamo yup great work! But please make sure to achieve some 60% of test coverage in this new module so we keep with a high level of testing. These days Python tests are preferred but yml will be accepted too.

**Comment by mileo** [comment]
@sadamo Por favor refaça o PR, fazendo um filterbranch para que os commits do Matheus e o Rafael Lima apareçam.

**Comment by sadamo** [comment]
Created new PR to add some initial commits: https://github.com/OCA/l10n-brazil/pull/420

**Pull request closed without merging**